File: algorithms&puzzles/prime.js
function isPrime(x) {
  if (x > 2) {
    for (var i = 2; i <= x-1; i++) {
      if (x % i === 0) {
        return false;
      }
    }
    return true;
  }
  else {
    return false;
  }
}

console.log(isPrime(13));

File: algorithms&puzzles/prime.py
def is_prime(x):
    if x <= 0:
        raise ValueError("You passed a negative number!")
    elif x > 1:
        for n in range(2, x):
            if x % n == 0:
                return False
        return True
    else:
        return False
print(is_prime(3))